### ISSUE TITLE
feedback mechanism from the CLI

### ISSUE THREAD

**Issue by amankalra172** [loophole/cli#13]
The idea here is to provide an option to take beta user to the feedback page. This would be useful to collect user data for feedback and understand how well appreciated is the application or if the user demands some new features. I will be creating a feedback form that will be added to our website. 

The feedback should be asked only after the user has finished using the tunneling service. For eg: when the user disconnects the tunneling service.. We could have a message saying : 

"Thank you for using Loophole. Please give us your feedback via *link* and help us improve our services. "

**Comment by Morishiri** [comment]
> I think we might need some sort of "Press Q to quit" functionality for this to work nicely. At the moment I close the CLI with CTRL+C and because of that, the deferred function in which I wanted to print the message isn't working.
> 
> EDIT: Alternatively we could capture the SIGINT and handle the shutdown after the user pressed CTRL+C ourselves, but I think having a specific key to close the program would be more user-friendly. Thoughts?

I think both should be supported.
